spec/requests/provider_management/external_users_spec.rb

```diff
@@ -1,14 +1,20 @@
 require 'rails_helper'
 
 RSpec.describe 'providers external users management', type: :request do
-  describe 'GET /provider_management/providers/:provider_id/external_users/:id/disable' do
-    subject(:disable_user) { get disable_provider_management_provider_external_user_path(provider, external_user) }
+  context 'when disabling the external_user' do
+    subject(:disable_user) do
+      patch update_availability_provider_management_provider_external_user_path(provider, external_user),
+            params: params
+    end
 
     let(:external_user) do
       create :external_user, provider: provider, user: create(:user, email: 'bubbletea@example.com')
     end
+
     let(:provider) { create :provider }
 
+    let(:params) { { external_user: { availability: 'false' } } }
+
```

Comment:
**(nit)** If the context is to disable the user (and likewise below, enable the user) you do not need to define another memoized value here, and the parameters could go straight in line 7.

Reply:
done